lib/runtime/wasmer/imports_test.go

```diff
@@ -409,6 +409,27 @@ func Test_ext_storage_set_version_1(t *testing.T) {
 	require.Equal(t, testvalue, val)
 }
 
+func Test_ext_offline_index_set_version_1(t *testing.T) {
+	// TODO this currently fails with error could nat find exported function, determine how else to test this
+	t.Skip()
+	inst := NewTestInstance(t, runtime.HOST_API_TEST_RUNTIME)
```

Comment:
what if change from `HOST_API_TEST_RUNTIME` to `NODE_RUNTIME`?

Reply:
I tried with `NODE_RUNTIME`, `NODE_RUNTIME_V098`, `POLKADOT_RUNTIME` and `DEV_RUNTIME` all with same results.  I think we need to build a updated test runtime that contains the "rtm_" imports that reflect to call the actual "ext" functions.

Reply:
@EclesioMeloJunior the `HOST_API_TEST_RUNTIME` is created by the w3f spec team, all it does is directly call the `ext_` function that is specified with the `rtm_` prefix. however it doesn't contain all the exports we need, so we might need to update it ourselves